code/modules/modular_computers/hardware/printer.dm

```diff
@@ -50,6 +50,42 @@
 		qdel(I)
 		stored_paper++
 		return TRUE
+	if(istype(I, /obj/item/paper_bin))
+		var/obj/item/paper_bin/bin = I
+		if(LAZYLEN(bin.papers))
+			if(stored_paper >= max_paper)
+				to_chat(user, span_warning("You try to feed \the [bin] into [src]'s paper recycler, but its paper bin is full!"))
+				return FALSE
+			/// Number of sheets we're adding
+			var/num_to_add = 0
+			/// Some goober put their manifesto in the paperbin, complain at the user if that's why nothing happened
+			var/rejected_sheets = FALSE
+			for(var/obj/item/paper/the_paper in bin.papers) // Search for the first blank sheet of paper, then toss it in
+				if(the_paper.info != "") // Uh oh, paper has words! 
+					rejected_sheets = TRUE
+					continue
+				if(istype(the_paper, /obj/item/paper/carbon)) // Add both the carbon, and the copy
+					var/obj/item/paper/carbon/carbon_paper = the_paper
+					if(!carbon_paper.copied && ((max_paper - stored_paper) >= 2)) // See if there's room for both
+						num_to_add = 2
+				else
+					num_to_add = 1
+				LAZYREMOVE(bin.papers, the_paper)
+				qdel(the_paper)
+				stored_paper += num_to_add
+				break // All full!
+			bin.update_appearance()
+			if(!num_to_add)
+				if(rejected_sheets)
+					to_chat(user, span_warning("The [src]'s paper recycler detects writing on everything in \the [bin], outright refusing to accept anything!"))
+				else
+					to_chat(user, span_warning("The [src]'s paper recycler refuses \the [bin], flashing some cryptic message about a feed error!"))
```

Comment:
PC load letter? What the hell does that mean??

Reply:
Basically that it needs paper. PC refers to paper cassette, load letter meaning to load letter-sized paper into the machine. A phrase thought up by engineers who dont speak human.